orc8r/gateway/c/common/sentry/SentryWrapper.cpp

```diff
@@ -105,7 +107,9 @@ void initialize_sentry(
     }
 
     sentry_init(options);
-
+    if (gethostname(node_name, sizeof(node_name)) == 0) {
+      sentry_set_tag(HOSTNAME, node_name);
+    }
```

Comment:
does this work?
```suggestion
    char node_name[HOST_NAME_MAX];
    if (gethostname(node_name, sizeof(HOST_NAME_MAX)) == 0) {
      sentry_set_tag(HOSTNAME, node_name);
    }
```

Reply:
sure it needs limits.h file let me add and push